### PR TITLE
Set focus border on header/footer/loanlist table headers

### LINKED ISSUE #337: Remove blue focus borders from components that appear on click
when does the border appear.
examples:
- loanlist when clicking on any row
- parent components when clicking on any link
- all buttons

### PR DESCRIPTION
**Pull request by acolytec3**
Partially fixed #337

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_ETHERSCAN_URL=https://kovan.etherscan.io
```
Comment (lucasvo) [inline .env:1]: @acolytec3 please remove

**Comment by acolytec3** [comment]
Do you want the whole `.env`  file removed or just the one etherscan link??

**Comment by ilinzweilin** [comment]
- Load investor details button still has border after clicking

**Comment by ilinzweilin** [comment]
@acolytec3 the whole file needs to go :)

**Pull request closed without merging**